<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-7R5CGPG" name="com.example.tests.ContactModificationTests" tests="5" failures="5" timestamp="23 Jan 2017 15:15:31 GMT" time="31.033" errors="0">
  <testcase name="modifySomeContact" time="6.365" classname="com.example.tests.ContactModificationTests">
    <failure type="java.lang.AssertionError" message="Lists differ at element [0]: ContactData [firstname=, lastname=contact-1741228021, address=contact-589587095, homephone=contact-669011528, mobilephone=contact354327450, workphone=contact619177266, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact828283162, secondaryphone=] != ContactData [firstname= contact-1741228021, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null] expected [ContactData [firstname=, lastname=contact-1741228021, address=contact-589587095, homephone=contact-669011528, mobilephone=contact354327450, workphone=contact619177266, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact828283162, secondaryphone=]] but found [ContactData [firstname= contact-1741228021, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null]]">
      <![CDATA[java.lang.AssertionError: Lists differ at element [0]: ContactData [firstname=, lastname=contact-1741228021, address=contact-589587095, homephone=contact-669011528, mobilephone=contact354327450, workphone=contact619177266, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact828283162, secondaryphone=] != ContactData [firstname= contact-1741228021, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null] expected [ContactData [firstname=, lastname=contact-1741228021, address=contact-589587095, homephone=contact-669011528, mobilephone=contact354327450, workphone=contact619177266, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact828283162, secondaryphone=]] but found [ContactData [firstname= contact-1741228021, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null]]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:551)
	at org.testng.Assert.assertEquals(Assert.java:516)
	at com.example.tests.ContactModificationTests.modifySomeContact(ContactModificationTests.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- modifySomeContact -->
  <testcase name="modifySomeContact" time="6.232" classname="com.example.tests.ContactModificationTests">
    <failure type="java.lang.AssertionError" message="Lists differ at element [49]: ContactData [firstname=contact-928493325, lastname=contact-1600198552, address=contact898387245, homephone=contact-132699873, mobilephone=contact-838118901, workphone=contact2004795078, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-712252228, secondaryphone=contact-1572203512] != ContactData [firstname=contact-928493325 contact-1600198552, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null] expected [ContactData [firstname=contact-928493325, lastname=contact-1600198552, address=contact898387245, homephone=contact-132699873, mobilephone=contact-838118901, workphone=contact2004795078, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-712252228, secondaryphone=contact-1572203512]] but found [ContactData [firstname=contact-928493325 contact-1600198552, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null]]">
      <![CDATA[java.lang.AssertionError: Lists differ at element [49]: ContactData [firstname=contact-928493325, lastname=contact-1600198552, address=contact898387245, homephone=contact-132699873, mobilephone=contact-838118901, workphone=contact2004795078, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-712252228, secondaryphone=contact-1572203512] != ContactData [firstname=contact-928493325 contact-1600198552, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null] expected [ContactData [firstname=contact-928493325, lastname=contact-1600198552, address=contact898387245, homephone=contact-132699873, mobilephone=contact-838118901, workphone=contact2004795078, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-712252228, secondaryphone=contact-1572203512]] but found [ContactData [firstname=contact-928493325 contact-1600198552, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null]]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:551)
	at org.testng.Assert.assertEquals(Assert.java:516)
	at com.example.tests.ContactModificationTests.modifySomeContact(ContactModificationTests.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- modifySomeContact -->
  <testcase name="modifySomeContact" time="6.022" classname="com.example.tests.ContactModificationTests">
    <failure type="java.lang.AssertionError" message="Lists differ at element [10]: ContactData [firstname=contact-111959760, lastname=contact1643959773, address=contact573242394, homephone=contact-291017077, mobilephone=contact1314277064, workphone=, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-1565248067, secondaryphone=contact1102568404] != ContactData [firstname=contact-111959760 contact1643959773, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null] expected [ContactData [firstname=contact-111959760, lastname=contact1643959773, address=contact573242394, homephone=contact-291017077, mobilephone=contact1314277064, workphone=, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-1565248067, secondaryphone=contact1102568404]] but found [ContactData [firstname=contact-111959760 contact1643959773, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null]]">
      <![CDATA[java.lang.AssertionError: Lists differ at element [10]: ContactData [firstname=contact-111959760, lastname=contact1643959773, address=contact573242394, homephone=contact-291017077, mobilephone=contact1314277064, workphone=, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-1565248067, secondaryphone=contact1102568404] != ContactData [firstname=contact-111959760 contact1643959773, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null] expected [ContactData [firstname=contact-111959760, lastname=contact1643959773, address=contact573242394, homephone=contact-291017077, mobilephone=contact1314277064, workphone=, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-1565248067, secondaryphone=contact1102568404]] but found [ContactData [firstname=contact-111959760 contact1643959773, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null]]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:551)
	at org.testng.Assert.assertEquals(Assert.java:516)
	at com.example.tests.ContactModificationTests.modifySomeContact(ContactModificationTests.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- modifySomeContact -->
  <testcase name="modifySomeContact" time="6.255" classname="com.example.tests.ContactModificationTests">
    <failure type="java.lang.AssertionError" message="Lists differ at element [78]: ContactData [firstname=contact723879568, lastname=contact-629191678, address=contact-2114324236, homephone=contact-2122757444, mobilephone=contact1659447436, workphone=contact901546339, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-2002338214, secondaryphone=contact2036231583] != ContactData [firstname=contact723879568 contact-629191678, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null] expected [ContactData [firstname=contact723879568, lastname=contact-629191678, address=contact-2114324236, homephone=contact-2122757444, mobilephone=contact1659447436, workphone=contact901546339, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-2002338214, secondaryphone=contact2036231583]] but found [ContactData [firstname=contact723879568 contact-629191678, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null]]">
      <![CDATA[java.lang.AssertionError: Lists differ at element [78]: ContactData [firstname=contact723879568, lastname=contact-629191678, address=contact-2114324236, homephone=contact-2122757444, mobilephone=contact1659447436, workphone=contact901546339, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-2002338214, secondaryphone=contact2036231583] != ContactData [firstname=contact723879568 contact-629191678, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null] expected [ContactData [firstname=contact723879568, lastname=contact-629191678, address=contact-2114324236, homephone=contact-2122757444, mobilephone=contact1659447436, workphone=contact901546339, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-2002338214, secondaryphone=contact2036231583]] but found [ContactData [firstname=contact723879568 contact-629191678, lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null]]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:551)
	at org.testng.Assert.assertEquals(Assert.java:516)
	at com.example.tests.ContactModificationTests.modifySomeContact(ContactModificationTests.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- modifySomeContact -->
  <testcase name="modifySomeContact" time="6.159" classname="com.example.tests.ContactModificationTests">
    <failure type="java.lang.AssertionError" message="Lists differ at element [36]: ContactData [firstname=contact-325665618, lastname=, address=contact306788595, homephone=contact452089784, mobilephone=contact1876851365, workphone=contact2036618770, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-1809099078, secondaryphone=contact1353026811] != ContactData [firstname=contact-325665618 , lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null] expected [ContactData [firstname=contact-325665618, lastname=, address=contact306788595, homephone=contact452089784, mobilephone=contact1876851365, workphone=contact2036618770, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-1809099078, secondaryphone=contact1353026811]] but found [ContactData [firstname=contact-325665618 , lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null]]">
      <![CDATA[java.lang.AssertionError: Lists differ at element [36]: ContactData [firstname=contact-325665618, lastname=, address=contact306788595, homephone=contact452089784, mobilephone=contact1876851365, workphone=contact2036618770, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-1809099078, secondaryphone=contact1353026811] != ContactData [firstname=contact-325665618 , lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null] expected [ContactData [firstname=contact-325665618, lastname=, address=contact306788595, homephone=contact452089784, mobilephone=contact1876851365, workphone=contact2036618770, email1=, emaill2=, bday=5, bmonth=May, byear=1985, secondaryaddress=contact-1809099078, secondaryphone=contact1353026811]] but found [ContactData [firstname=contact-325665618 , lastname=null, address=null, homephone=null, mobilephone=null, workphone=null, email1=null, emaill2=null, bday=null, bmonth=null, byear=null, secondaryaddress=null, secondaryphone=null]]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:496)
	at org.testng.Assert.assertEquals(Assert.java:125)
	at org.testng.Assert.assertEquals(Assert.java:551)
	at org.testng.Assert.assertEquals(Assert.java:516)
	at com.example.tests.ContactModificationTests.modifySomeContact(ContactModificationTests.java:41)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:354)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:312)
	at org.testng.SuiteRunner.run(SuiteRunner.java:261)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1215)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1140)
	at org.testng.TestNG.run(TestNG.java:1048)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- modifySomeContact -->
</testsuite> <!-- com.example.tests.ContactModificationTests -->
